app/src/main/res/layout/activity_uo.xml

```diff
@@ -1,34 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:context="com.myskng.virtualchemlight.Activity.UOActivity">
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
+            xmlns:tools="http://schemas.android.com/tools"
+            xmlns:app="http://schemas.android.com/apk/res-auto"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:background="@android:color/black">
+            tools:context="com.myskng.virtualchemlight.activity.UOActivity">
 
-        <ImageView
+        <FrameLayout
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/ce428e626934c566136cc7a36e8a65181a716a26/app/src/main/res/layout/activity_uo.xml#L11"></span>This view is not constrained. It only has designtime positions, so it will jump to (0,0) at runtime unless you add the constraints</td>
    </tr>
  </tbody>
</table>